# HG changeset patch
# User Júlio Hoffimann Mendes <julio.hoffimann@gmail.com>
# Date 1385222962 28800
#      Sat Nov 23 08:09:22 2013 -0800
# Node ID edaa9ed49a1f0abf65a4e4822708577b14857e76
# Parent  2c1f14991bcc0a9d95261cb6b50ca585eb0150c3
pt_BR.ts: Update Octave Portugese/Brazil translation.

* pt_BR.ts: Update Octave Portugese/Brazil translation.

diff --git a/libgui/languages/pt_BR.ts b/libgui/languages/pt_BR.ts
--- a/libgui/languages/pt_BR.ts
+++ b/libgui/languages/pt_BR.ts
@@ -115,23 +115,23 @@
     <message>
         <location line="-269"/>
         <source>Octave Files (*.m);;All Files (*)</source>
         <translation>Scripts do Octave (*.m);;Todos os Arquivos (*)</translation>
     </message>
     <message>
         <location line="-40"/>
         <source>New Function</source>
-        <translation type="unfinished"></translation>
+        <translation>Nova Função</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>New function name:
 </source>
-        <translation type="unfinished"></translation>
+        <translation>Novo nome de função:</translation>
     </message>
     <message>
         <location line="+281"/>
         <source>File not saved! A file with the selected name
 %1
 is already open in the editor</source>
         <translation>Arquivo não salvo! Um arquivo com o nome selecionado %1 já está aberto no editor</translation>
     </message>
@@ -213,57 +213,57 @@ is already open in the editor</source>
     <message>
         <location line="+3"/>
         <source>&amp;Remove All Bookmarks</source>
         <translation>&amp;Remover Todos os Marcadores</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>&amp;Next Breakpoint</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Próximo ponto de parada</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Pre&amp;vious Breakpoint</source>
-        <translation type="unfinished"></translation>
+        <translation>Ponto de Parada &amp;Anterior</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Toggle &amp;Breakpoint</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Ativar/desativar Ponto de Parada</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Remove All Breakpoints</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Remover todos Pontos de Parada</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Comment</source>
         <translation>&amp;Comentar</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Uncomment</source>
         <translation>&amp;Descomentar</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Save File and Run</source>
-        <translation type="unfinished"></translation>
+        <translation>Salvar Arquivo e Rodar</translation>
     </message>
     <message>
         <location line="+57"/>
         <source>&amp;Recent Editor Files</source>
         <translation>Arquivos &amp;Recentes</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>New &amp;Function</source>
-        <translation type="unfinished"></translation>
+        <translation>Nova &amp;Função</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>&amp;Edit Function</source>
         <translation>&amp;Editar Função</translation>
     </message>
     <message>
         <location line="+10"/>
@@ -453,22 +453,22 @@ foi removido ou renomeado. Pretende salvá-lo agora?%2</translation>
         <source>Could not open file %1 for write:
 %2.</source>
         <translation>Não foi possível abrir arquivo %1 para escrita:
 %2.</translation>
     </message>
     <message>
         <location line="-962"/>
         <source>Line:</source>
-        <translation type="unfinished"></translation>
+        <translation>Linha:</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Col:</source>
-        <translation type="unfinished"></translation>
+        <translation>Coluna:</translation>
     </message>
     <message>
         <location line="+1134"/>
         <source>It seems that &apos;%1&apos; has been modified by another application. Do you want to reload it?</source>
         <translation>Parece que o arquivo &apos;%1&apos; foi modificado por outra aplicação. Deseja recarregá-lo?</translation>
     </message>
 </context>
 <context>
@@ -544,37 +544,37 @@ foi removido ou renomeado. Pretende salvá-lo agora?%2</translation>
         <location line="-360"/>
         <location line="+363"/>
         <source>New Directory</source>
         <translation>Novo Diretório</translation>
     </message>
     <message>
         <location line="-323"/>
         <source>Double-click a file to open it</source>
-        <translation type="unfinished"></translation>
+        <translation>Clique duas vezes em um arquivo para abrí-lo</translation>
     </message>
     <message>
         <location line="+236"/>
         <source>File size</source>
-        <translation type="unfinished"></translation>
+        <translation>Tamanho do arquivo</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>File type</source>
-        <translation type="unfinished"></translation>
+        <translation>Tipo do arquivo</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Date modified</source>
-        <translation type="unfinished"></translation>
+        <translation>Data de modificação</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Show hidden</source>
-        <translation type="unfinished"></translation>
+        <translation>Mostrar escondidos</translation>
     </message>
     <message>
         <location line="+24"/>
         <source>Open</source>
         <translation>Abrir</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -674,57 +674,72 @@ para:</translation>
         <translation>Criar diretório em</translation>
     </message>
 </context>
 <context>
     <name>final_page</name>
     <message>
         <location filename="../src/welcome-wizard.cc" line="+194"/>
         <source>Enjoy!</source>
-        <translation type="unfinished"></translation>
+        <translation>Aproveite!</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Previous</source>
-        <translation type="unfinished"></translation>
+        <translation>Anterior</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Finish</source>
-        <translation type="unfinished">Finalizar</translation>
+        <translation>Finalizar</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Cancel</source>
-        <translation type="unfinished">Cancelar</translation>
+        <translation>Cancelar</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>&lt;html&gt;&lt;body&gt;
 &lt;p&gt;We hope you find Octave to be a useful tool.&lt;/p&gt;
 &lt;p&gt;If you encounter problems, there are a number of ways to get help including commercial support options, a mailing list, a wiki, and other commnity-based support channels.
 You can find more information about each of these by visiting &lt;a href=&quot;http://octave.org/support.html&quot;&gt;http://octave.org/support.html&lt;/a&gt; (opens in external browser).&lt;/p&gt;
 &lt;/body&gt;&lt;/html&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;html&gt;&lt;body&gt;
+&lt;p&gt;Esperamos que o Octave seja uma ferramenta últil.&lt;/p&gt;
+&lt;p&gt;Se encontrar problemas, existem várias maneiras de pedir ajuda incluindo suporte comercial, lista de email, wiki, e outros canais de suporte da comunidade.
+Você pode encontrar mais informações sobre cada um deles visitando &lt;a href=&quot;http://octave.org/support.html&quot;&gt;http://octave.org/support.html&lt;/a&gt; (opens in external browser).&lt;/p&gt;
+&lt;/body&gt;&lt;/html&gt;</translation>
     </message>
     <message>
         <location line="+21"/>
         <source>&lt;html&gt;&lt;head&gt;
 &lt;style&gt;
 a:link { text-decoration: underline; color: #0000ff; }
 &lt;/style&gt;
 &lt;head/&gt;&lt;body&gt;
 &lt;p&gt;For more information about Octave:&lt;/p&gt;
 &lt;ul&gt;
 &lt;li&gt;Visit &lt;a href=&quot;http://octave.org&quot;&gt;http://octave.org&lt;/a&gt; (opens in external browser)&lt;/li&gt;
 &lt;li&gt;Get the documentation online as &lt;a href=&quot;http://www.gnu.org/software/octave/doc/interpreter/index.html&quot;&gt;html&lt;/a&gt;- or &lt;a href=&quot;http://www.gnu.org/software/octave/octave.pdf&quot;&gt;pdf&lt;/span&gt;&lt;/a&gt;-document (opens in external browser)&lt;/li&gt;
 &lt;li&gt;Open the documentation browser of the Octave GUI with the help menu&lt;/li&gt;
 &lt;/ul&gt;
 &lt;/body&gt;&lt;/html&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;html&gt;&lt;head&gt;
+&lt;style&gt;
+a:link { text-decoration: underline; color: #0000ff; }
+&lt;/style&gt;
+&lt;head/&gt;&lt;body&gt;
+&lt;p&gt;Para mais informações sobre o Octave:&lt;/p&gt;
+&lt;ul&gt;
+&lt;li&gt;Visite &lt;a href=&quot;http://octave.org&quot;&gt;http://octave.org&lt;/a&gt; (opens in external browser)&lt;/li&gt;
+&lt;li&gt;Acesse a documentação online como documento &lt;a href=&quot;http://www.gnu.org/software/octave/doc/interpreter/index.html&quot;&gt;html&lt;/a&gt;- ou &lt;a href=&quot;http://www.gnu.org/software/octave/octave.pdf&quot;&gt;pdf&lt;/span&gt;&lt;/a&gt; (abre em navegador externo)&lt;/li&gt;
+&lt;li&gt;Abrir a documentação na interface gráfica com o menu de ajuda&lt;/li&gt;
+&lt;/ul&gt;
+&lt;/body&gt;&lt;/html&gt;</translation>
     </message>
 </context>
 <context>
     <name>find_dialog</name>
     <message>
         <location filename="../src/m-editor/find-dialog.cc" line="+77"/>
         <source>Find &amp;what:</source>
         <translation>&amp;Buscar por:</translation>
@@ -986,17 +1001,17 @@ a:link { text-decoration: underline; col
     <message>
         <location filename="../src/history-dock-widget.cc" line="+43"/>
         <source>Browse and search the command history.</source>
         <translation>Naveguar e buscar no histórico de comandos.</translation>
     </message>
     <message>
         <location line="+24"/>
         <source>Double-click a command to transfer it to the terminal.</source>
-        <translation type="unfinished"></translation>
+        <translation>Clique duas vezes em um comando para transferí-lo ao terminal.</translation>
     </message>
     <message>
         <location line="+9"/>
         <source>Enter text to filter the command history.</source>
         <translation>Digite para filtrar o hitórico de comandos.</translation>
     </message>
     <message>
         <location line="+4"/>
@@ -1019,36 +1034,40 @@ a:link { text-decoration: underline; col
         <translation>Criar script</translation>
     </message>
 </context>
 <context>
     <name>initial_page</name>
     <message>
         <location filename="../src/welcome-wizard.cc" line="-179"/>
         <source>Welcome to Octave!</source>
-        <translation type="unfinished">Bem-vindo ao Octave!</translation>
+        <translation>Bem-vindo ao Octave!</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Next</source>
-        <translation type="unfinished"></translation>
+        <translation>Próximo</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Cancel</source>
-        <translation type="unfinished">Cancelar</translation>
+        <translation>Cancelar</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>&lt;html&gt;&lt;body&gt;
 &lt;p&gt;You seem to be using the Octave graphical interface for the first time on this computer.
 Click &apos;Next&apos; to create a configuration file and launch Octave.&lt;/p&gt;
 &lt;p&gt;The configuration file is stored in %1.  If that file exists, you will not see this dialog when Octave starts.&lt;/p&gt;
 &lt;/body&gt;&lt;/html&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;html&gt;&lt;body&gt;
+&lt;p&gt;Parece que você está usando a interface gráfica pela primeira vez neste computador.
+Clique em próximo para criar um arquivo de configuração e lançar o Octave.&lt;/p&gt;
+&lt;p&gt;O arquivo de configuração é armazenado em %1. Se o arquivo existir, você não verá esta mensagem quando o Octave inicializar.&lt;/p&gt;
+&lt;/body&gt;&lt;/html&gt;</translation>
     </message>
 </context>
 <context>
     <name>main_window</name>
     <message>
         <location filename="../src/main-window.cc" line="+201"/>
         <location line="+1149"/>
         <source>Load Workspace</source>
@@ -1139,17 +1158,17 @@ Click &apos;Next&apos; to create a confi
     <message>
         <location line="+13"/>
         <source>Octave Release Notes</source>
         <translation>Notas da Versão do Octave</translation>
     </message>
     <message>
         <location line="+185"/>
         <source>Octave Community News</source>
-        <translation type="unfinished"></translation>
+        <translation>Novidades da Comunidade</translation>
     </message>
     <message>
         <location line="+171"/>
         <source>Set working directory</source>
         <translation>Definir diretório de trabalho</translation>
     </message>
     <message>
         <location line="+754"/>
@@ -1204,22 +1223,22 @@ Click &apos;Next&apos; to create a confi
     <message>
         <location line="+26"/>
         <source>File Browser</source>
         <translation>Navegador de Arquivos</translation>
     </message>
     <message>
         <location line="-94"/>
         <source>Step In</source>
-        <translation type="unfinished"></translation>
+        <translation>Passo a dentro</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Step Out</source>
-        <translation type="unfinished"></translation>
+        <translation>Passo a fora</translation>
     </message>
     <message>
         <location line="+108"/>
         <source>Reset Default Window Layout</source>
         <translation>Restaurar Disposição de Janelas Padrão</translation>
     </message>
     <message>
         <location line="+77"/>
@@ -1364,34 +1383,57 @@ Octave&apos;s community news source seem
 &lt;/p&gt;
 &lt;p&gt;
 For the latest news, please check
 &lt;a href=&quot;http://octave.org/community-news.html&quot;&gt;http://octave.org/community-news.html&lt;/a&gt;
 when you have a connection to the web (link opens in an external browser).
 &lt;/p&gt;
 &lt;p&gt;
 &lt;small&gt;&lt;em&gt;&amp;mdash; The Octave Developers, </source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;html&gt;
+&lt;body&gt;
+&lt;p&gt;
+Fonte de notícias da comunidade parece estar indisponível.
+&lt;/p&gt;
+&lt;p&gt;
+Para as últimas notícias, por favor cheque
+&lt;a href=&quot;http://octave.org/community-news.html&quot;&gt;http://octave.org/community-news.html&lt;/a&gt;
+quando tiver conexão com a internet (abre em navegador externo).
+&lt;/p&gt;
+&lt;p&gt;
+&lt;small&gt;&lt;em&gt;&amp;mdash; Os desenvolvdores do Octave, </translation>
     </message>
     <message>
         <location line="+18"/>
         <source>&lt;html&gt;
 &lt;body&gt;
 &lt;p&gt;
 Connecting to the web to display the latest Octave Community news has been disabled.
 &lt;/p&gt;
 &lt;p&gt;
 For the latest news, please check
 &lt;a href=&quot;http://octave.org/community-news.html&quot;&gt;http://octave.org/community-news.html&lt;/a&gt;
 when you have a connection to the web (link opens in an external browser)
 or enable web connections for news in Octave&apos;s network settings dialog.
 &lt;/p&gt;
 &lt;p&gt;
 &lt;small&gt;&lt;em&gt;&amp;mdash; The Octave Developers, </source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;html&gt;
+&lt;body&gt;
+&lt;p&gt;
+Conectar a web para mostrar as últimas notícias da comunidade foi desativado.
+&lt;/p&gt;
+&lt;p&gt;
+Para as últimas notícias, por favor cheque
+&lt;a href=&quot;http://octave.org/community-news.html&quot;&gt;http://octave.org/community-news.html&lt;/a&gt;
+quando tiver conexão com a web (abre em navegador externo)
+ou ative as conexões web para obter notícias em configurações de rede do Octave.
+&lt;/p&gt;
+&lt;p&gt;
+&lt;small&gt;&lt;em&gt;&amp;mdash; Os desenvolvedores do Octave, </translation>
     </message>
 </context>
 <context>
     <name>octave_dock_widget</name>
     <message>
         <location filename="../src/octave-dock-widget.cc" line="+61"/>
         <location line="+163"/>
         <source>Undock widget</source>
@@ -1433,36 +1475,36 @@ or enable web connections for news in Oc
 </context>
 <context>
     <name>octave_qt_link</name>
     <message>
         <location filename="../src/octave-qt-link.cc" line="+94"/>
         <location line="+6"/>
         <location line="+7"/>
         <source>Yes</source>
-        <translation type="unfinished"></translation>
+        <translation>Sim</translation>
     </message>
     <message>
         <location line="-13"/>
         <source>No</source>
-        <translation type="unfinished"></translation>
+        <translation>Não</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>File
 %1
 does not exist. Do you want to create it?</source>
-        <translation type="unfinished">Arquivo
+        <translation>Arquivo
 %1
 não existe. Deseja criá-lo?</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Octave Editor</source>
-        <translation type="unfinished">Editor do Octave</translation>
+        <translation>Editor do Octave</translation>
     </message>
     <message>
         <location line="+211"/>
         <source>The file %1 does not exist in the load path.  To debug the function you are editing, you must either change to the directory %2 or add that directory to the load path.</source>
         <translation>O arquivo %1 não existe no caminho de pesquisa.  Para depurar a função que você está editando, é preciso ou mudar para o diretório %2 ou adicionar esse diretório ao caminho de pesquisa.</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -1496,22 +1538,28 @@ não existe. Deseja criá-lo?</translation>
         <location filename="../src/resource-manager.cc" line="+204"/>
         <source>The settings file
 %1
 does not exist and can not be created.
 Make sure you have read and write permissions to
 %2
 
 Octave GUI must be closed now.</source>
-        <translation type="unfinished"></translation>
+        <translation>O arquivo de configuração
+%1
+não existe e não pode ser criado.
+Tenha certeza de que você tem permissão de leitura/escrita em
+%2
+
+Octave GUI precisa ser fechado agora.</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Octave Critical Error</source>
-        <translation type="unfinished"></translation>
+        <translation>Erro crítico no Octave</translation>
     </message>
 </context>
 <context>
     <name>settings_dialog</name>
     <message>
         <location filename="../src/settings-dialog.ui" line="+29"/>
         <source>Settings</source>
         <translation>Configurações</translation>
@@ -1639,47 +1687,47 @@ Octave GUI must be closed now.</source>
     <message>
         <location line="-282"/>
         <source>Show complete path in window title</source>
         <translation>Exibir caminho completo no título da janela</translation>
     </message>
     <message>
         <location line="-17"/>
         <source>Show whitespace</source>
-        <translation type="unfinished"></translation>
+        <translation>Mostrar espaço em branco</translation>
     </message>
     <message>
         <location line="+27"/>
         <source>Do not show whitespace used for indentation</source>
-        <translation type="unfinished"></translation>
+        <translation>Não mostrar espaço em branco usado para identação</translation>
     </message>
     <message>
         <location line="+290"/>
         <source># of characters typed before completion list displayed</source>
-        <translation type="unfinished"></translation>
+        <translation>Número de caracteres digitados antes da lista de completação ser apresentada</translation>
     </message>
     <message>
         <location line="+194"/>
         <source>Create nonexistent files without prompting</source>
-        <translation type="unfinished"></translation>
+        <translation>Criar arquivos não existentes sem perguntar</translation>
     </message>
     <message>
         <location line="+57"/>
         <source>command line (%f=file, %l=line):</source>
-        <translation type="unfinished"></translation>
+        <translation>linha de comando (%f=arquivo, %l=linha):</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>emacs</source>
         <translation>emacs</translation>
     </message>
     <message>
         <location line="+36"/>
         <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select font, font size (as a difference from the default size), font style (bold, italic, underline), text color and background color (for the latter, the color pink (255,0,255) is a placeholder for the default background color)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Selecionar fonte, tamanho da fonte (como uma diferença do tamanho padrão), estilo da fonte (negrito, itálico, sublinhado), cor de texto e cor de fundo (a cor rosa (255,0,255) é a cor de fundo padrão)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
     </message>
     <message>
         <location line="+31"/>
         <source>Terminal</source>
         <translation>Terminal</translation>
     </message>
     <message>
         <location line="+15"/>
@@ -1689,17 +1737,17 @@ Octave GUI must be closed now.</source>
     <message>
         <location line="+23"/>
         <source>Cursor blinking</source>
         <translation>Indicador piscante</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Use foreground color</source>
-        <translation type="unfinished"></translation>
+        <translation>Usar cor de frente</translation>
     </message>
     <message>
         <location line="+94"/>
         <source>Font size</source>
         <translation>Tamanho de fonte</translation>
     </message>
     <message>
         <location line="+35"/>
@@ -1744,17 +1792,17 @@ Octave GUI must be closed now.</source>
     <message>
         <location line="+35"/>
         <source>Network</source>
         <translation>Rede</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Allow Octave to connect to the Octave web site to display current news and information</source>
-        <translation type="unfinished"></translation>
+        <translation>Permitir Octave se conectar a web para mostrar notícias recentes e informação</translation>
     </message>
     <message>
         <location line="+46"/>
         <source>Use proxy server</source>
         <translation>Usar servidor proxy</translation>
     </message>
     <message>
         <location line="-23"/>
@@ -1774,32 +1822,32 @@ Octave GUI must be closed now.</source>
     <message>
         <location line="+7"/>
         <source>Icon size</source>
         <translation>Tamanho do Ícone</translation>
     </message>
     <message>
         <location line="+991"/>
         <source>Synchronize Octave working directory with file browser</source>
-        <translation type="unfinished"></translation>
+        <translation>Sincronizar diretório de trabalho com o navegador de arquivos</translation>
     </message>
     <message>
         <location line="+129"/>
         <source>Socks5Proxy</source>
         <translation>Proxy Socks 5</translation>
     </message>
     <message>
         <location line="-16"/>
         <source>Hostname:</source>
         <translation>Nome do servidor:</translation>
     </message>
     <message>
         <location line="+44"/>
         <source>Proxy type:</source>
-        <translation type="unfinished"></translation>
+        <translation>Tipo de Proxy:</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Port:</source>
         <translation>Porta:</translation>
     </message>
     <message>
         <location line="-27"/>
@@ -1859,53 +1907,64 @@ Octave GUI must be closed now.</source>
         <translation>u</translation>
     </message>
 </context>
 <context>
     <name>setup_community_news</name>
     <message>
         <location filename="../src/welcome-wizard.cc" line="+44"/>
         <source>Community News</source>
-        <translation type="unfinished">Novidades da Comunidade</translation>
+        <translation>Novidades da Comunidade</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Previous</source>
-        <translation type="unfinished"></translation>
+        <translation>Anterior</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Next</source>
-        <translation type="unfinished"></translation>
+        <translation>Próximo</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Cancel</source>
-        <translation type="unfinished">Cancelar</translation>
+        <translation>Cancelar</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>&lt;html&gt;&lt;body&gt;
 &lt;p&gt;When the Octave GUI starts, it will check the Octave web site for current news and information about the Octave community.
 The check will happen at most once each day and news will only be displayed if there is something new since the last time you viewed the news.&lt;/p&gt;
 &lt;p&gt;You may also view the news by selecting the &quot;Community News&quot; item in the &quot;Help&quot; menu in the GUI, or by visiting
 &lt;a href=&quot;http://octave.org/community-news.html&quot;&gt;http://octave.org/community-news.html&lt;/a&gt;.&lt;/p&gt;
 &lt;/body&gt;&lt;/html&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;html&gt;&lt;body&gt;
+&lt;p&gt;Quando o Octave GUI inicia, ele checa o site para conferir notícias recentes e da comunidade.
+A checagem acontece pelo menos uma vez ao dia e as notícias só serão apresentadas se existir algo novo desde o último acesso.&lt;/p&gt;
+&lt;p&gt;Você também pode ver as notícias em &quot;Notícias de Comunidade&quot; no menu de ajuda na interface gráfica, ou visitar
+&lt;a href=&quot;http://octave.org/community-news.html&quot;&gt;http://octave.org/community-news.html&lt;/a&gt;.&lt;/p&gt;
+&lt;/body&gt;&lt;/html&gt;</translation>
     </message>
     <message>
         <location line="+26"/>
         <source>&lt;html&gt;&lt;head&gt;
 &lt;style&gt;
 a:link { text-decoration: underline; color: #0000ff; }
 &lt;/style&gt;
 &lt;head/&gt;&lt;body&gt;
 &lt;p&gt;Allow Octave to connect to the Octave web site when it starts to display current news and information about the Octave community.&lt;/p&gt;
 &lt;/body&gt;&lt;/html&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;html&gt;&lt;head&gt;
+&lt;style&gt;
+a:link { text-decoration: underline; color: #0000ff; }
+&lt;/style&gt;
+&lt;head/&gt;&lt;body&gt;
+&lt;p&gt;Permitir Octave se conectar ao web na inicialização para mostrar as notícias correntes e informações sobre a comunidade.&lt;/p&gt;
+&lt;/body&gt;&lt;/html&gt;</translation>
     </message>
 </context>
 <context>
     <name>terminal_dock_widget</name>
     <message>
         <location filename="../src/terminal-dock-widget.cc" line="+38"/>
         <source>Command Window</source>
         <translation>Janela de Comandos</translation>
@@ -1963,17 +2022,17 @@ a:link { text-decoration: underline; col
         <location line="+109"/>
         <source>Right click to copy, rename, or display</source>
         <translation>Clique com botão direito para copiar, renomear ou exibir</translation>
     </message>
     <message>
         <location line="+37"/>
         <location line="+2"/>
         <source>complex</source>
-        <translation type="unfinished"></translation>
+        <translation>complexo</translation>
     </message>
 </context>
 <context>
     <name>workspace_view</name>
     <message>
         <location filename="../src/workspace-view.cc" line="+47"/>
         <source>Workspace</source>
         <translation>Ambiente de trabalho</translation>
